course_catalog/management/commands/backpopulate_ocw_next_data.py

```diff
@@ -46,7 +46,8 @@ def handle(self, *args, **options):
                     )
                 )
                 runs = LearningResourceRun.objects.filter(
-                    slug=course_url_substring, platform=PlatformType.ocw.value
+                    slug="courses/{}".format(course_url_substring),
+                    platform=PlatformType.ocw.value,
```

Comment:
**Suggestion:** I think it would be worthwhile to update the parameter description, and possible the name.

IMO:
```diff
         parser.add_argument(
-            "--course-url-substring",
-            dest="course_url_substring",
+            "--course-name",
+            dest="course_name",
             required=False,
-            help="If set, backpopulate only courses whose urls match with this substring",
+            help="If set, backpopulate only the course with this name",
         )
```

- `name` is the OCW term for these things.
- The old text says "courses that match...". It's only ever at most 1 course, right?

Reply:
No it can be multiple names seperated by a comma

Reply:
I'll add that to the documentation